test/e2e/framework/resource_usage_gatherer.go

```diff
@@ -370,6 +371,29 @@ const (
 	MasterAndDNSNodes NodesSet = 2
 )
 
+// isNodeHaveControlPlanePods returns true if specified node has control plane pods
+// (kube-scheduler and/or kube-controller-manager).
+func isNodeHaveControlPlanePods(c clientset.Interface, nodeName string) (bool, error) {
```

Comment:
i think we should call it `NodeHasControlPlanePods()`
`has` is the right singular form, while `is` and `has/have` are conflicting verbs.

Reply:
Nice point.
OK, I will rename this function.